whois-commons/src/main/resources/whois.properties

```diff
@@ -136,3 +136,5 @@ rdap.redirect.apnic=http://rdap.apnic.net/
 rdap.redirect.afrinic=
 rdap.redirect.arin=http://rdappilot.arin.net
 rdap.redirect.lacnic=http://restfulwhoisv2.labs.lacnic.net
+
+client.cert.auth.enabled=true
```

Comment:
Be careful with this, I think Spring will default to TRUE if it's not explicitly set to FALSE in the external properties file (and what's the order?). Is it safer to remove it?

Reply:
Yes, it's gone, well spotted!